Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
-Add information here
+
+
+**What is SLI and SLO?**
+Reliability is the concept of a system having the quality of being trustworthy or of performing consistently well. Today most applications have a Service level objective (SLO) to facilitate monitoring. SLOs have typically 3 constraints
+1)	Service level indicator or SLI.
+2)	The target metric or Objective in percentile 
+3)	The observation window
+SLI + Objective +Observation window = SLO
```

Comment:
I think it might make this clearer to not use mathematical notation of "+" in here unless each of the components have the same units and are compatible mathematically to yield the final value of SLO vs. them just forming components such that they might have different units, etc. but are to be combined to yield the final value of the SLO.